middleware/notFound.tsx

```diff
@@ -15,15 +15,11 @@ export const handleNotFound: Middleware = async (ctx, next) => {
       const page = renderSSR(
         <App>
           <ErrorBody title="Not Found">
-            The requested URL ({`<code>"${ctx.request.url}"</code>`}) was not
-            found.
+            The requested URL <code>{ctx.request.url.href}</code> was not found.
```

Comment:
Not sure about this change, but I guess we shouldn't display `code` tag in the error message.
![Uploading Screen Shot 2022-05-07 at 23.36.07.png…]()